packages/engine/lib/error/src/report.rs

```diff
@@ -19,27 +21,27 @@ use crate::{
 /// the [`Backtrace` documentation][`Backtrace`]. To enable the span trace, [`ErrorLayer`] has to
 /// be enabled.
 ///
-/// Context information can be added by using [`wrap()`] or [`ResultExt`]. The [`Frame`] stack can
-/// be iterated by using [`frames()`].
+/// Context information can be added by using [`attach_message()`] or [`ResultExt`]. The [`Frame`]
```

Comment:
I think this phrasing is a bit confusing. We should find out how to phrase our (Message/Provider/Context) things better in plain English. Perhaps we just leave that for a final clean-up PR pre-release, can you make an Asana task (to tidy up the doc-strings for phrasing of (Message/Provider/Context)?

Reply:
[Asana task](https://app.asana.com/0/0/1202371933275761/f) (_internal_)